external/fv3fit/fv3fit/reservoir/domain.py

```diff
@@ -8,3 +16,213 @@ def slice_along_axis(arr: np.ndarray, inds: slice, axis: int = 0):
     sl = [slice(None)] * arr.ndim
     sl[axis] = inds
     return arr[tuple(sl)]
+
+
+class CubedsphereDivider:
```

Comment:
As long as this class is only used by `save_ranks.py`, I would suggest putting it in that file or at least in that directory. I don't see this needing to be re-used by fv3fit as written, because you preprocess data with this before passing it to fv3fit.

Reply:
nit: Elsewhere we've always called it `CubedSphere`